codebuild/samples/linux-smoke-tests.yml

```diff
@@ -16,10 +16,6 @@ phases:
       - echo Build started on `date`
       - $CODEBUILD_SRC_DIR/codebuild/samples/setup-linux.sh
       - $CODEBUILD_SRC_DIR/codebuild/samples/pubsub-linux.sh
-      - $CODEBUILD_SRC_DIR/codebuild/samples/connect-websocket-linux.sh
```

Comment:
Do we need codebuild at all anymore?

Reply:
We do not need it for running samples, but there are webhooks for running Codebuild that we cannot remove without removing it for the entire repository. I also need to look at migrating Codebuild as well in the future, so I'm leaving just the `PubSub` Codebuild for that step in the migration.